GGProject/workflow/tests.py

```diff
@@ -1,3 +1,48 @@
 from django.test import TestCase
+from django.urls import reverse
+from rest_framework.test import APITestCase, APIClient
+from rest_framework.views import status
+from .models import Employee
+from .serializers import EmployeesSerializer
 
-# Create your tests here.
+
+# Creates API test case
+class BaseViewTest(APITestCase):
+
+    client = APIClient()
+
+    # Method for creating employees
+    @staticmethod
+    def create_employee(first_name="", last_name="", emp_ID=None, emp_email = "", emp_permissions =None, manager_email = "", key_code = None):
+        if first_name != "" and last_name != "" and  emp_ID != None and emp_permissions != None, and manager_email != "" and key_code != None:
+            Employee.objects.create(first_name=first_name, last_name=last_name, emp_ID=emp_ID, emp_email = emp_email, emp_permissions = emp_permissions, manager_email = manager_email, key_code = key_code)
```

Comment:
These couple lines should also be consistent with spacing around the `=`. It just helps with readability

Reply:
The tests.py file should actually just be cleared out. The tests listed in it don't work with our current models. I can go ahead and update it.